DigitalLearningSolutions.Web/Views/MyAccount/EditDetails.cshtml

```diff
@@ -26,29 +28,81 @@
+    </div>
+  </div>
+
   <div class="nhsuk-grid-row">
     <div class="nhsuk-grid-column-full">
-      <vc:text-input asp-for="FirstName" label="First Name" type="text" spell-check="false" autocomplete="given-name" />
+      <vc:text-input asp-for="Email" label="Email address" type="text" spell-check="false" autocomplete="email" hint-text="" css-class="nhsuk-u-width-one-half" />
     </div>
   </div>
 
   <div class="nhsuk-grid-row">
     <div class="nhsuk-grid-column-full">
-      <vc:text-input asp-for="LastName" label="Last Name" type="text" spell-check="false" autocomplete="family-name" />
+      <div class="divider"></div>
+    </div>
+  </div>
+
+  <div class="nhsuk-grid-row">
+    <div class="nhsuk-grid-column-one-half">
+      @{
+        var hintText = @"To change your profile picture, select a new image and click the Preview button to preview it.
+                              If you upload a new profile image and do not preview it before saving, then your profile picture
+                              will not be updated.";
+      }
+      <input type="hidden" asp-for="NewProfileImage" />
+      <input type="hidden" asp-for="CurrentProfileImage" />
+      <vc:file-input asp-for="ProfilePicture" label="Profile picture (optional)" hint-text="@hintText" css-class="nhsuk-u-width-full" />
+    </div>
+    <div class="nhsuk-grid-column-one-half">
+      <input type="hidden" asp-for="HasProfileImageBeenRemoved" />
+      @if (!Model.HasProfileImageBeenRemoved)
+      {
+        <div class="profile-picture__image-box">
+          @if (Model.NewProfileImage != null)
+          {
+            <img class="profile-picture__image" src="data:image;base64,@Convert.ToBase64String(Model.NewProfileImage)" alt="Profile Picture" />
+          }
+          else if (Model.CurrentProfileImage != null)
+          {
+            <img class="profile-picture__image" src="data:image;base64,@Convert.ToBase64String(Model.CurrentProfileImage)" alt="Profile Picture" />
+          }
+        </div>
+      }
+    </div>
+  </div>
+
+  <div class="nhsuk-grid-row">
+    <div class="nhsuk-grid-column-one-half">
+      <button name="action" class="nhsuk-button" value="previewImage">Preview</button>
+      <button name="action" class="nhsuk-button nhsuk-button--secondary" value="removeImage">Remove</button>
     </div>
   </div>
 
   <div class="nhsuk-grid-row">
     <div class="nhsuk-grid-column-full">
-      <vc:text-input asp-for="Email" label="Email Address" type="text" spell-check="false" autocomplete="email" />
+      <div class="divider"></div>
     </div>
   </div>
 
   <div class="nhsuk-grid-row">
     <div class="nhsuk-grid-column-full">
-      <vc:text-input asp-for="Password" label="Confirm password to save changes:" type="password" spell-check="false" autocomplete="email" />
+      <vc:text-input asp-for="Password" label="Confirm password to save changes:" type="password" spell-check="false" autocomplete="" hint-text="" css-class="nhsuk-u-width-one-half" />
```

Comment:
Ah, not a big fan of the empty hint-text etc fields.. Apparently it's only fixed in .NET 5 😢